metrics/prometheus.go

```diff
@@ -1045,7 +1045,30 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					return metricValues{{value: float64(s.Processes.FdCount), timestamp: s.Timestamp}}
 				},
 			},
+			{
+				name:      "container_pids_max",
+				help:      "The maximum number of processes allowed in the container, infinity if value is zero",
+				valueType: prometheus.GaugeValue,
+				getValues: func(s *info.ContainerStats) metricValues {
+					return metricValues{{
+						value:     float64(s.Processes.ThreadsMax),
+						timestamp: s.Timestamp,
+					}}
+				},
+			},
+			{
+				name:      "container_pids_current",
```

Comment:
`container_threads`.  To be consistent with other metrics, omit the `_current` suffix.